VideoLocker/res/layout/registration_spinner_item.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
nit: Is this spinner registration specific, or can we make this the general edX spinner layout?